armi/reactor/grids/tests/test_grids.py

```diff
@@ -467,6 +468,47 @@ def test_adjustPitch(self):
             for i in range(3):
                 self.assertEqual(grid.offset[i], offset)
 
+    def test_adjustPitchCornersUp(self):
+        """Adjust the pich of a hexagonal lattice, for a "corners up" grid.
+
+        .. test:: Construct a hexagonal lattice with three rings.
+            :id: T_ARMI_GRID_HEX1
+            :tests: R_ARMI_GRID_HEX
+
+        .. test:: Return the grid coordinates of different locations.
+            :id: T_ARMI_GRID_GLOBAL_POS1
+            :tests: R_ARMI_GRID_GLOBAL_POS
+        """
+        # run this test for a grid with no offset, and then a few random offset values
+        for offset in [0, 1, 1.123, 3.14]:
+            offsets = [offset, 0, 0]
+            # build a hex grid with pitch=1, 3 rings, and the above offset
+            grid = grids.HexGrid(
+                unitSteps=(
+                    (0.5, -0.5, 0),
+                    (1.5 / math.sqrt(3), 1.5 / math.sqrt(3), 0),
+                    (0, 0, 0),
+                ),
+                unitStepLimits=((-3, 3), (-3, 3), (0, 1)),
+                offset=numpy.array(offsets),
+                cornersUp=True,
+            )
+
+            # test that we CAN change the pitch, and it scales the grid (but not the offset)
+            v1 = grid.getCoordinates((1, 0, 0))
+            grid.changePitch(2.0)
+            self.assertAlmostEqual(grid.pitch, math.sqrt(3), delta=1e-9)
+            v2 = grid.getCoordinates((1, 0, 0))
+            correction = numpy.array([0.5, math.sqrt(3) / 2, 0])
+            assert_allclose(v1 + correction, v2)
+
+            # basic sanity: test number of rings has changed
```

Comment:
I do not follow this comment. The number of rings _has_ changed?

Reply:
That's a typo I copied from the original test. Let me fix the typo and prove that it hasn't change with an extra assert. One sec.

Reply:
Done